native/mod_manager/mod_manager.c

```diff
@@ -925,6 +925,32 @@ static apr_status_t mod_manager_manage_worker(request_rec *r, nodeinfo_t *node,
     return balancer_manage(r, params);
 }
 
+/*
+ * Check if the proxy balancer module already has a worker
+ * and return the id
+ */
+static proxy_worker *proxy_node_getid(request_rec *r, nodeinfo_t *nodeinfo, int *id, proxy_server_conf **the_conf)
+{
+    if (balancerhandler != NULL) {
+        return (balancerhandler->proxy_node_getid(r, nodeinfo->mess.balancer, nodeinfo->mess.Type, nodeinfo->mess.Host, nodeinfo->mess.Port, id, the_conf));
```

Comment:
The outer parentheses are not needed. Also, maybe adding check for `nodeinfo` being `NULL` is not a bad idea.

Reply:
Same a above a separated PR please.